core/sql/executor/ExHbaseAccess.h

```diff
@@ -342,10 +342,10 @@ class ExHbaseAccessTcb  : public ex_tcb
   void setRowID(char *rowId, Lng32 rowIdLen);
   void allocateDirectBufferForJNI(UInt32 rowLen);
   void allocateDirectRowBufferForJNI(short numCols, 
-                          short maxRows = 1);
+                          UInt16 maxRows = 1);
```

Comment:
This can't be changed to Unsigned because this number is sent to Java. There is no unsigned concept in java. Basically, we can't send more than 32767 rows in one buffer.